ilc/common/wrapWithCache.js

```diff
@@ -36,11 +52,24 @@ const wrapWithCache = (localStorage, logger, createHash = hashFn) => (fn, cacheP
                     cachedAt: now,
                 }));
 
+                logger.info({
+                    memoName: name,
+                    hash,
+                    now,
+                    cachedAt: parsedItem ? parsedItem.cachedAt : undefined
+                }, logMessage('Item read from API. Invalidation finished.'));
+
                 return JSON.parse(localStorage.getItem(hash));
             }).catch(err => {
                 delete cacheResolutionPromise[hash];
 
                 if (localStorage.getItem(hash) === null) {
+                    logger.info({
```

Comment:
I'd use error here

Reply:
I also think in that way but want to analyze log on raw examples. There are some places in this logic I disagree with.